src/main/java/repository/InteractionWithDB.java

```diff
@@ -0,0 +1,116 @@
+package repository;
+
+import model.CarEntity;
+
+import java.sql.*;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.logging.Logger;
+
+public class InteractionWithDB {
+    Logger logger = Logger.getLogger(getClass().getName());
+    public Connection connect(String dataName, String user, String pass){
+        Connection conn = null;
+        try {
+            conn = DriverManager.getConnection("jdbc:postgresql://localhost:5432/" + dataName, user, pass);
+            System.out.println("Connection Established");
+            System.out.println();
+
+        } catch (Exception e) {
+            System.out.println("Connection Failed");
+            System.out.println(e);
+        }
+        return conn;
+    }
+    public void createCarsTable(Connection conn, String tableName) throws SQLException {
```

Comment:
jak przerobisz tą klasę jak napisałem wyżej to wtedy będziesz miał pole Connection conn, jako private final klasy, więc w tych wszystkich metodach usuniesz go z argumentów metod